docker/sql_validate.sh

```diff
@@ -3,8 +3,11 @@
 SERVICE=$1
 
 # validate mysql
-echo "sleeping for 60 seconds during mysql boot..."
-sleep 60
+echo "Checking every 3s for mysql to be ready"
```

Comment:
Yay!

Reply:
according to @zarsky-broad it'll reduce to 10 sec after this change :)